pkg/actor/system_test.go

```diff
@@ -128,15 +128,16 @@ func TestRouterSendAndSendB(t *testing.T) {
 	require.Equal(t, context.Canceled, <-ch)
 }
 
-func wait(t *testing.T, timeout time.Duration, f func()) {
+func wait(t *testing.T, f func()) {
 	wait := make(chan int)
 	go func() {
 		f()
 		wait <- 0
 	}()
 	select {
 	case <-wait:
-	case <-time.After(timeout):
+	case <-time.After(5 * time.Second):
```

Comment:
why use the hardcoded time here?  is it better to extend the timeout on the function call?

Reply:
The `wait` is specialized in actor testing, 5 seconds are longer enough for most cases, we can pass timeout when it's needed in future.